templates/project/create.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base_headless.html" %}
+{% load i18n %}
+
+{% block title %}<title>Palamar | Create Project </title>{% endblock title %}
```

Comment:
This can be variable send by the view function. I have used title variable on my other view functions and passed to template render and called the variable with {{ title }}.

If you think this is the right way. We need to use {{ trans }} to translate "Create Project"